geonode_mapstore_client/client/js/routes/Home.jsx

```diff
@@ -435,6 +438,9 @@ function Home({
                                 query={query}
                                 onChange={isSmallDevice && handleUpdateSmallDevice || handleUpdate}
                                 onClose={handleShowFilterForm}
+                                isSmallDevice={isSmallDevice}
+                                submitOnChangeField={filterFormsSubmitOnChange}
```

Comment:
We should avoid to make the FiltersForm aware the FilterForm component of the size of the device in addition it seems the isSmallDevice is used to check if submitOnChangeField should be true or false.
Could we use `submitOnChangeField={!isSmallDevice}` instead and remove the `isSmallDevice` prop?

Reply:
two props, isSmallDevice and submitOnChangeField, to have the option to configure the form submit, even for desktop solutions not only for mobile.  If we don't need this option for the moment, and the apply button is for mobile only, I can remove the conf and props